### PR TITLE
Update Chinese translations for string.xml

### PR DESCRIPTION
**Pull request by Tribbtsz**

### CODE REVIEW

**Comment by Wqttzicue** [comment]
Something on Pingpong language..

File: app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -393,22 +406,25 @@
...
         <item quantity="other">从黑名单中移除所有 %1$s 首歌曲</item>
     </plurals>
 
     <plurals name="format_reset_quick_picks_amount">
+    	<item quantity="one">删除 %1$s 次播放事件</item>
         <item quantity="other">删除 %1$s 次播放事件</item>
     </plurals>
 
     <plurals name="song_count_plural">
+    	<item quantity="one">%1$d 首歌曲</item>
```
Comment (25huizengek1) [inline app/src/main/res/values-zh/strings.xml:427]: Why did you add these quantities? According to Lint these are redundant. Aren't they exactly the same?

**Review by 25huizengek1 — CHANGES_REQUESTED** [review]
LGTM, but this minor warning needs to be resolved.
(If Android Lint is wrong here, we should add `tools:ignore="UnusedQuantity"` and move on.)

File: app/src/main/res/values-zh/strings.xml (hunk 1 above)
Comment (Tribbtsz) [reply app/src/main/res/values-zh/strings.xml]: Ah I see - you're right, I mistakenly added those plural quantities when they weren't needed. Sorry for the confusion, I'll remove them in the next update.

**Review by 25huizengek1 — APPROVED** [review]
Great, thank you for the contribution!
